htdocs/class/xoopsform/formdhtmltextarea.php

```diff
@@ -197,7 +222,13 @@ public function codeIcon()
                 $this->js .= $js;
             }
         }
-        $code .= "<img src='" . XOOPS_URL . "/images/code.gif' alt='" . _XOOPS_FORM_ALT_CODE . "' title='" . _XOOPS_FORM_ALT_CODE . "' onclick='xoopsCodeCode(\"{$textarea_id}\", \"" . htmlspecialchars(_ENTERCODE, ENT_QUOTES) . "\");'  onmouseover='style.cursor=\"hand\"'/>&nbsp;" . "<img src='" . XOOPS_URL . "/images/quote.gif' alt='" . _XOOPS_FORM_ALT_QUOTE . "' title='" . _XOOPS_FORM_ALT_QUOTE . "' onclick='xoopsCodeQuote(\"{$textarea_id}\", \"" . htmlspecialchars(_ENTERQUOTE, ENT_QUOTES) . "\");' onmouseover='style.cursor=\"hand\"'/>";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $code .= "<button type='button' class='btn btn-default btn-xs' onclick='xoopsCodeCode(\"{$textarea_id}\", \"" . htmlspecialchars(_ENTERCODE, ENT_QUOTES) . "\");' onmouseover='style.cursor=\"hand\"' title='" . _XOOPS_FORM_ALT_CODE . "'><span class='glyphicon glyphicon-menu-left' aria-hidden='true'></span><span class='glyphicon glyphicon-menu-right' aria-hidden='true'></span> " . _XOOPS_FORM_ALT_CODE . "</button>&nbsp;";
+            $code .= "<button type='button' class='btn btn-default btn-xs' onclick='xoopsCodeQuote(\"{$textarea_id}\", \"" . htmlspecialchars(_ENTERQUOTE, ENT_QUOTES) . "\");' onmouseover='style.cursor=\"hand\"' title='" . _XOOPS_FORM_ALT_QUOTE . "'><span class='glyphicon glyphicon-bullhorn' aria-hidden='true'></span> " . _XOOPS_FORM_ALT_QUOTE . "</button>&nbsp;";
+            $code .= "</div></div>";
+        } else {
```

Comment:
__[phpmd]__
* The method codeIcon uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->